### PR TITLE
lift-mysql-discovery-script

### PR DESCRIPTION
**Pull request by Soumyadeep-Oracle**

### CODE REVIEW

File: cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/files/MySQLdiscovery.sh
Comment (AlexanderHodicke) [inline cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/files/MySQLdiscovery.sh:1]: @oheimburger code review

**Review by AlexanderHodicke — CHANGES_REQUESTED** [review]

File: cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# MySQL Database Discovery Utility
+
+MySQL Database Discovery & Analysis Utility is a discovery utility to fetch the database information and the host details. In order to capture complete details of the source MySQL instance, the discovery script will help us fetch the necessary information. It is a shell script with HTML tags embedded to help connect to source instances and fetch the details. This script runs in any flavor of Linux version and any MySQL version 5.6 and above.
```
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/README.md:3]: Replace '&' with 'and'.

Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/README.md]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/files/MySQLdiscovery.sh [hunk 2]
```diff
@@ -0,0 +1,174 @@
+#############################################################
+--# Copyright (c) 2023, Oracle and/or its affiliates.
+--# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+--# Purpose: Discovery Script for MySQL Databases. 
+--# This script is to be executed at the discovery phase before planning for migration/upgrade.
+--# Version  :  1.1
+--# USAGE    :  sh <filename.sh> .
+--# @author: Ajay Rao.
+##############################################################
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/mysql-discovery-utility/files/MySQLdiscovery.sh]: Submitted with required changes.

File: cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/LICENSE
Comment (oheimburger) [inline cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/LICENSE:1]: What is this file for?

Comment (Soumyadeep-Oracle) [reply cloud-architecture/custom-apps-and-consolidation/oracle-db-consolidation/db-discovery/discovery-utility/LICENSE:1]: This LICENSE file was missing for an earlier asset uploaded. The asset location was discovery-utility/files. As the LICENSE was missed earlier, uploaded the same to correct the same.

**Review by AlexanderHodicke — APPROVED** [review]

**Pull request merged**